### ISSUE TITLE
how to obtain the NO.case and NO.control

### ISSUE THREAD

**Issue by linnanqia** [xinhe-lab/mirage#3]
Dear,
       I still felt confused about how to obtain the NO.case and NO.control. You said that " (No.case, how many times the variant appears in cases, No.contr, how many times the variant appears in controls — you can compute these quantities from your data)". which file I can get this information. Can you give me an example?Thank you!

**Comment by han16** [comment]
No.case and No.contr are total number of alleles in cases and controls at a genomic locus. These numbers are typically very small in rare variant analysis. Hope this helps.

**Comment by linnanqia** [comment]
Hi， dear，
        Is it true to obtain the NO.case and NO.control?
#NO.case
#grep 0/1
count_case_01 <- data.frame(apply(tmp_vcf_case_data,1,function(x) length(grep('0/1',x))))
rownames(count_case_01) <- tmp_vcf_case_data[,1]
colnames(count_case_01) <- "count_01"
#grep 1/2
count_case_12 <- data.frame(apply(tmp_vcf_case_data,1,function(x) length(grep('1/2',x))))
rownames(count_case_12) <- tmp_vcf_case_data[,1]
colnames(count_case_12) <- "count_12"


I am waiting for your response!  Many Thanks!







--

谭海珠
Haizhu Tan      
汕头大学医学院
Shantou University Medical School 






在 2021-03-13 04:54:25，"Shengtong" ***@***.***> 写道：

No.case and No.contr are total number of alleles in cases and controls at a genomic locus. These numbers are typically very small in rare variant analysis. Hope this helps.

**Comment by han16** [comment]
I am not sure about your sample but the data is passed along to me from bioinformaticians.
